sbc_compassion/models/correspondence.py

```diff
@@ -54,6 +54,10 @@ class Correspondence(models.Model):
     name = fields.Char(compute='_set_name')
     correspondant_id = fields.Many2one(
         related='sponsorship_id.correspondant_id', store=True)
+    redactor_id = fields.Many2one(
+        'res.partner', 'Partner', store=True,
+        readonly=False, compute=lambda self:
```

Comment:
Should not be a computed field. The user can simply select an encourager if needed.